app/controllers/agents/sessions_controller.rb

```diff
@@ -1,6 +1,19 @@
 class Agents::SessionsController < Devise::SessionsController
   before_action :exclude_signed_in_users, only: [:new]
 
+  def new
+    # Le flash d'erreur est trop aggressif pour le cas d'un agent non connecté.
```

Comment:
```suggestion
    # Le flash d'erreur est trop agressif pour le cas d'un agent non connecté.
```

bonjour je suis monsieur pointilleux 

mais très cool ces commentaires, ça clairife bien l’intention 👍